_posts/2015-12-24-the-future-of-crystal.md

```diff
@@ -3,6 +3,7 @@ title: The future of Crystal
 summary: A short story
 thumbnail: 🎄
 author: asterite
+categories: project
```

Comment:
missing tags

Reply:
Tags are just added as a byproduct and have no effect. I don't think it makes much sense to strive for completeness here.